coalib/output/printers/LogPrinter.py

```diff
@@ -1,54 +1,161 @@
+import sys
 import traceback
+import logging
+from colorama import Style
 
```

Comment:
It seems to comment on the very wrong line. Don't know if this issue is already reported. @sils1297